.github/workflows/niployments.yml

```diff
@@ -0,0 +1,23 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Upload to NIployments register
+      uses: NIAEFEUP/push-to-niployments@v2.1
+      with:
+        project_name: nimentas-sasup
+        repository_name: website
```

Comment:
Oh, right, I just remembered something!
@Naapperas @LuisDuarte1 can you please confirm these values are correct?

From what I've seen in the action code/docs, the project_name should be `niaefeup` or am I missing something?

Reply:
@thePeras @rubuy-74 as well, since this is a question of how the harbor registry is set up.

Reply:
xD This is an Harbor naming thingy

Basically in Harbor a project is a collection of repositories, which in turn are a collection of tagged images. I believe @LuisDuarte1 wanted the default project to be `niaefeup` so that's what I left as the default (trivia: `v1` had a different naming scheme) and made the repository name the project name (with this coming from either action input or the GitHub repository's name).

You can check the behavior [here](https://github.com/NIAEFEUP/push-to-niployments/blob/75a84da1816c1a5703c07b409a2b2571384b802e/action.yaml#L48).

Reply:
Basically this will store the project under `<registry url>/<project>/<repository>`

Reply:
Regarding their correctness, unless you want to specify custom values for them, you should leave the fields blank.

Reply:
Awesome. What repository should I use? `niaefeup`?

Reply:
One good example of how this naming scheme would (normally) be applied is with projects that have both a FrontEnd and a BackEnd: taking NIJobs as an example, you could have a `nijobs` project in Harbor and this project would have `fe` and `be` repositories, each containing tagged images build from the action and the respective GitHub repositories.

Reply:
I see... That was my initial intuition with using ementas/website. But I'm guessing we can also use niaefeup/nijobs-fe and niaefeup/nijobs-be...

Reply:
Yeah, we're currently using niaefeup/nijobs-fe and niaefeup/nijobs-be (https://github.com/NIAEFEUP/niployments/blob/main/services%2Fni%2Fnijobs%2F02-deployments.yaml#L18). This project will follow the convention, then!

Reply:
Yeah, that is the plan.

Opinions differ in this matter: I personally am in favor of each of NIAEFEUP's projects having a separate Harbor project.for now everything is stored under `niaefeup` with the Harbor repository name being the GitHub repository's name, unless explicitely specified. In this case, the final builds will be stored under `niaefeup/nimentas-sasup`. 

CC @LuisDuarte1 on this one :)